--- php-7.1.24/scripts/phpize.in.phpize	2018-11-11 07:43:21.000000000 +0000
+++ php-7.1.24/scripts/phpize.in	2018-11-11 07:53:41.000000000 +0000
@@ -162,6 +162,15 @@
   $PHP_AUTOHEADER || exit 1
 }
 
+phpize_check_headers()
+{
+  if test ! -f $includedir/main/php.h; then
+    echo "Can't find PHP headers in $includedir"
+    echo "The php-devel package is required for use of this command."
+    exit 1
+  fi
+}
+
 # Main script
 
 case "$1" in
@@ -180,12 +189,15 @@
 
   # Version
   --version|-v)
+    phpize_check_headers
     phpize_print_api_numbers
     exit 0
   ;;
 
   # Default
   *)
+     phpize_check_headers
+
      phpize_check_configm4 0
 
      phpize_check_build_files
